Source/include/Ably/ARTPush.h

```diff
@@ -33,6 +33,13 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)didAblyPushRegistrationFail:(nullable ARTErrorInfo *)error;
 
+/**
+ Ably will call the implementation of this method when the activation process is completed with success, so you can request other types of device tokens, such as `location`, `pushtotalk` etc.
+ See https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/sending_notification_requests_to_apns for possible values of `apns-push-type`.
+ Before making request to Apple for alternative device token you should check whether you've already did this after application launch.
```

Comment:
> you should check whether you've already did this after application launch

Why?

Reply:
Please, see my  response to [here](https://github.com/ably/ably-cocoa/pull/1771#discussion_r1261804835).

Reply:
Which part of that answers my question? I wanted to know:

> you should check whether you've already did this

Done what? And what if I have?

> after application launch

What's application launch got to do with anything? The Ably push notification registration flow isn't related to application launch. Yes, I'm sure there are plenty of apps that initiate this flow upon application launch, but that's not necessarily always the case.

Reply:
https://github.com/ably/ably-cocoa/pull/1771#discussion_r1264776018